From f073493ebc279cefa391d364c48265058795e1d2 Mon Sep 17 00:00:00 2001
From: Martin Liska <mliska@suse.cz>
Date: Mon, 23 May 2022 13:44:39 +0200
Subject: [PATCH] Fix building with GCC 13.

Fixes the following issue:
base64.hpp:23:49: error: 'uint8_t' was not declared in this scope

and:
deprecation.hpp:20:22: error: found ':' in nested-name-specifier, expected '::'
---
 src/deprecation.hpp          | 1 +
 src/serialization/base64.hpp | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/deprecation.hpp b/src/deprecation.hpp
index 33736df1b22c..ee09e73be1d0 100644
--- a/src/deprecation.hpp
+++ b/src/deprecation.hpp
@@ -14,6 +14,7 @@
 
 #pragma once
 
+#include <cstdint>
 #include <string>
 
 /** See https://wiki.wesnoth.org/CompatibilityStandards for more info. */
diff --git a/src/serialization/base64.hpp b/src/serialization/base64.hpp
index dbb3b3d32652..c5d8dd32d3fd 100644
--- a/src/serialization/base64.hpp
+++ b/src/serialization/base64.hpp
@@ -15,6 +15,7 @@
 
 #pragma once
 
+#include <cstdint>
 #include <string_view>
 #include <vector>
 
